symplyphysics/laws/thermodynamics/volumetric_and_linear_expansion_coefficients_in_isotropic_materials.py

```diff
@@ -21,7 +26,39 @@
 
 law = Eq(volumetric_expansion_coefficient, 3 * linear_expansion_coefficient)
 
-# TODO: Derive from their definitions for a cube-sized object
+# Derive from their definitions for a cube-sized object
+
+_volume_change = symbols("volume_change")
+_initial_length, _length_change = symbols("initial_length length_change")
+_temperature_change = symbols("temperature_change")
+
+_final_length_expr = _initial_length + _length_change
+_initial_volume_expr = _initial_length**3
+_final_volume_expr = _final_length_expr**3
+
+_volumetric_expr = volumetric_def.definition.rhs.subs({
+    volumetric_def.volume(volumetric_def.temperature).diff(volumetric_def.temperature): _volume_change / _temperature_change,
+    _volume_change: _final_volume_expr - _initial_volume_expr,
+    volumetric_def.volume(volumetric_def.temperature): _initial_volume_expr,
```

Comment:
Not justified. It assumes `V(t)` is `V(t0)` which I suggested above.

Reply:
I'll get back to you on this

Reply:
And about this part here, even if we choose `_final_volume_expr` to be the denominator, it is the same as `_initial_volume_expr` in the `_volume_change -> 0` limit. Or, $`\frac{1}{V + \Delta V} = \frac{1}{V} \frac{1}{1 + \frac{\Delta V}{V}} = \frac{1}{V} \left(1 + \sum_{i \ge 1} \left( - \frac{\Delta V}{V} \right)^i \right) = \frac{1}{V}`$ in the limit $`\Delta V \to 0`$.

Reply:
Still we should somehow justify how we choose this parameter. You can either add some explanation, or add condition that `volumetric_def.volume(temperature) == linear_def.length(temperature)**3` and derive from it.

Reply:
Sure